frontend/index.html.ejs

```diff
@@ -39,15 +14,51 @@
 
   <script type="text/javascript">
     <% if (htmlWebpackPlugin.options.production) { %>
-    window['NOMAD_ENDPOINT'] = window.location.host;
+    window['NOMAD_PRELOAD_ENDPOINT'] = (window.location.host + window.location.pathname).replace(/\/$/, '');
     <% } else { %>
-    window['NOMAD_ENDPOINT'] = window.location.hostname + ':' + (NOMAD_ENDPOINT_PORT || 3000);
+    window['NOMAD_PRELOAD_ENDPOINT'] = window.location.hostname + ':' + (NOMAD_ENDPOINT_PORT || 3000);
     <% } %>
-
     var configScript = document.createElement('script');
-    configScript.setAttribute('src', '//' + NOMAD_ENDPOINT + '/config.js');
+    configScript.setAttribute('src', '//' + NOMAD_PRELOAD_ENDPOINT + '/config.js');
+    configScript.onload = function () {
```

Comment:
Yeah, this could be an artifact of my troubleshooting. I'll re-tool and see what happens.

I think I recall using relative paths to have no effect a while ago, at least when was trying to get the webpack-generated content to load. I will give this a shot - it would indeed be much simpler and I can move the various content points back to their respective places versus an `onload` parameter.